Moesif.Api/Http/Client/UnirestClient.cs

```diff
@@ -46,6 +47,16 @@ public class UnirestClient: IHttpClient
         static UnirestClient() {
             SharedClient = new UnirestClient();
             Version = Assembly.GetExecutingAssembly().FullName;
+            InitClientDefaults(httpClient, Configuration.BaseUri);
+        }
```

Comment:
We will see if you can dependency injection today to improve DNS resolution and SSL handshake time.